### PR TITLE
fix(example): update demo example

### PR DESCRIPTION
**Pull request by dackerman**
Updates `demo.ts` under examples to use the updated workers AI client code.

### CODE REVIEW

**Review by rattrayalex — APPROVED** [review]

File: examples/demo.ts [hunk 1]
```diff
@@ -6,9 +6,8 @@ import Cloudflare from 'cloudflare';
 const client = new Cloudflare();
 
 async function main() {
-  const accountId = '<your accound ID>';
-
-  const response = await client.ai.run(accountId, '@cf/meta/llama-2-7b-chat-int8', {
+  const response = await client.workers.ai.run('@cf/meta/llama-2-7b-chat-int8', {
+    account_id: '<your account ID>',
```
Comment (rattrayalex) [inline examples/demo.ts:10]:
```suggestion
    account_id: process.env['CLOUDFLARE_ACCOUNT_ID'],
```

unless there's an account ID that cloudflare is comfortable exposing publicly as a demo ID? @jacobbednarz ?

Comment (jacobbednarz) [reply examples/demo.ts:10]: in the demos, i'd rather lean on environment variables and checks to allow people to copy/paste them and know they work as they should.

account/zone ids aren't sensitive so i can add the example to GitHub Actions later today for this to work.

File: examples/demo.ts
Comment (jacobbednarz) [inline examples/demo.ts:1]: are we able to structure this directory into specific examples? i.e. `examples/workers/ai/demo.ts` for flexibility as this grows?

**Comment by jacobbednarz** [comment]
TIL about `#!/usr/bin/env -S npm run tsn -T`, nice one!

File: examples/demo.ts
Comment (dackerman) [reply examples/demo.ts:1]: yeah we can do that!

**Review by jacobbednarz — APPROVED** [review]

File: examples/demo.ts
Comment (rattrayalex) [reply examples/demo.ts:1]: (in this case, could rename it to `workers-ai-demo.ts`)

**Pull request merged**